### ISSUE TITLE
Babel plugin attempts to (?) evaluate electron sources

### ISSUE THREAD

**Issue by Kelin2025** [callstack/linaria#461]
## Environment

```
"electron": "^5.0.6"
"linaria": "^1.3.1"
```

## Description

Sometimes build fails with the following error:

```
⠧ Building index.ts.../Users/kelin/Work/shelter-app/node_modules/elect
ron/index.js:7
  if (fs.existsSync(pathFile)) {
         ^

TypeError: Cannot read property 'existsSync' of null
    at getElectronPath (/Users/kelin/Work/shelter-app/node_modules/electron/index.js:7:10)
    at /Users/kelin/Work/shelter-app/node_modules/electron/index.js:18:18
    at Script.runInContext (vm.js:74:29)
    at Module.evaluate (/Users/kelin/Work/shelter-app/node_modules/linaria/lib/babel/module.js:203:12)
    at Module.<anonymous> (/Users/kelin/Work/shelter-app/node_modules/linaria/lib/babel/module.js:184:13)
    at Module.require (/Users/kelin/Work/shelter-app/node_modules/linaria/lib/babel/module.js:134:23)
    at /Users/kelin/Work/shelter-app/ui/atoms/ExternalLink.tsx:10:17
    at Script.runInContext (vm.js:74:29)
    at Module.evaluate (/Users/kelin/Work/shelter-app/node_modules/linaria/lib/babel/module.js:203:12)
    at Module.<anonymous> (/Users/kelin/Work/shelter-app/node_modules/linaria/lib/babel/module.js:184:13)
🚨  /Users/kelin/Work/shelter-app/frontend/app/features/options/moleculas/OptionsField.tsx: Cannot set property 'fileName' of undefined
    at Pipeline.process (/Users/kelin/Work/shelter-app/node_modules/parcel/src/Pipeline.js:31:22)
```
Error stack refers to this code:
![image](https://user-images.githubusercontent.com/4208480/61406660-f6e93800-a8e4-11e9-83ed-61f886bd4524.png)

And here is what it tries to evaluate:
![image](https://user-images.githubusercontent.com/4208480/61406711-0ec0bc00-a8e5-11e9-8ef0-01305497fa2d.png)

## Reproducible Demo

Honestly, I can't figure out what causes the issue
Here's my babelrc:
```
{
  "presets": [
    "@babel/preset-react",
    [
      "@babel/preset-env",
      {
        "modules": "commonjs",
        "targets": {
          "node": "8"
        }
      }
    ],
    "@babel/preset-typescript",
    "linaria/babel"
  ],
  "plugins": [
    "inline-dotenv",
    "@babel/plugin-proposal-class-properties",
    [
      "module-resolver",
      {
        "root": ["/"],
        "alias": {
          "common": "./common",
          "lib": "./lib",
          "migrations": "./migrations",
          "ui": "./ui",
          "env": "./env"
        }
      }
    ]
  ]
}
```
And, also I use Parcel for build

**Comment by brandonkal** [comment]
The fs module is not permitted for preval. Also note that process.env = { NODE_ENV }. No other environment variable is available in the sandbox.

**Comment by Kelin2025** [comment]
Yes but this is `electron` sources, not mine

```
at getElectronPath (/Users/kelin/Work/shelter-app/node_modules/electron/index.js:7:10)
```

**Comment by brandonkal** [comment]
I see. The issue then is that linaria's shaker only runs once per babel entrypoint. This can pose a problem if you are importing anything through an index module (and it also slows down compilation significantly).

I am currently re-writing the preval support in my fork which should address your issue.

**Comment by jayu** [comment]
Any reproduction repo would be appreciated 😊

**Comment by jayu** [comment]
I will reopen this if any reproduction will be provided. I believe issues of that kind will be solved in [`2.0`](https://github.com/callstack/linaria/milestone/7) release due to a new evaluation strategy that should shake unused imports.
